routes/lang/en-application.json

```diff
@@ -146,7 +146,12 @@
   "app.lastname": "Last name",
   "app.email": "Email",
   "app.validationrequired": "A value is required here",
-  "app.validationlength": "At least %{length} characters are required",
+  "app.validationpasswordlength": "At least %{length} characters are required",
+  "app.validationpassworddigits": "Password must include at least %{length} digit(s)",
+  "app.validationpasswordlowercase": "Password must include at least %{length} lower case character(s)",
+  "app.validationpassworduppercase": "Password must include at least %{length} upper case character(s)",
+  "app.validationpasswordspecial": "Password must include at least %{length} non alpha nummeric character(s)",
```

Comment:
Typo `non alpha nummeric` → `non-alphanumeric`.

Also, should we be defining the characters from the regex here?